Tensile/LibraryIO.py

```diff
@@ -264,7 +264,10 @@ def _getLibraryLogicForSchedule(self, schedulePrefix, architectureName, deviceNa
       tileSelectionIndices = logicTuple[6]
       tileSelectionLogic["TileSelectionIndices"] = tileSelectionIndices
       data.append(tileSelectionLogic)
+    else:
+      data.append(None)
 
```

Comment:
Are we prepared to handle "None" objects in parsing?

Reply:
As far as I can tell, yes. It ends up coming out as `null` in the yaml file. Having the else branch here ensures that `logicTuple[7]` (the Benchmark/PerformanceMetric) ends up at a consistent index in the library logic file.

All the local testing I did follows the else branch and works. What I haven't tested is having tile selection indices _and_ a BenchmarkMetric, although I can't think of any reasons why this wouldn't work.